src/Passwordless/Passwordless.csproj

```diff
@@ -45,11 +50,6 @@
     <PackageReference Include="System.Net.Http.Json" Version="7.0.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="../../favicon.png" Pack="true" PackagePath="/" Visible="false" />
-    <None Include="../../README.md" Pack="true" PackagePath="/" Visible="false" />
-  </ItemGroup>
```

Comment:
Why is this removed while it seems to be kept in the AspNetCore csproj?

Reply:
I actually just moved it to the top of the file for consistency. It's still there.